proto/src/main/proto/Rss.proto

```diff
@@ -26,6 +26,7 @@ package rss.common;
 service ShuffleServer {
   rpc registerShuffle (ShuffleRegisterRequest) returns (ShuffleRegisterResponse);
   rpc unregisterShuffle(ShuffleUnregisterRequest) returns (ShuffleUnregisterResponse);
+  rpc unregisterApp(AppUnregisterRequest) returns (AppUnregisterResponse);
```

Comment:
Prefer using `unregisterShuffleByAppId(String appId);`
Because we don't have registerApp interface. It's weird to use interface `unregisterApp`.

Reply:
done.